dask_glm/logistic.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from scipy.optimize import fmin_l_bfgs_b
+import dask.array as da
+
+
+def logistic_regression(X, y, alpha, rho, over_relaxation):
+    N = 5
```

Comment:
N is the number of equal sized chunks. This needs to be calculated with the dask array.